Server/JokesOnYou.Web.Api/Startup.cs

```diff
@@ -36,13 +37,17 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<ITokenService, JwtTokenService>();
             services.AddScoped<IUserRepository, UserRepository>();
             services.AddScoped<IUnitOfWork, UnitOfWork>();
+            services.AddScoped<IJokesRepository, JokesRepository>();
+            services.AddScoped<IJokesService, JokesService>();
 
             services.AddMvc()
             .AddJsonOptions(o => {
```

Comment:
not sure when i missed this, could you explain? (if you wrote it)

Reply:
woeps that should have been commented out in a fix earlier back hmm ill remove it again.